examples/nlp/squad.py

```diff
@@ -0,0 +1,330 @@
+"""
+Copyright 2018 The Google AI Language Team Authors and
+The HuggingFace Inc. team.
+Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Some transformer of this code were adapted from the HuggingFace library at
+https://github.com/huggingface/transformers
+
+Download the Squad data by running the script:
+examples/nlp/scripts/download_squad.py
+
+To finetune Squad v1.1 on pretrained BERT large uncased on 1 GPU:
+python squad.py
+--data_dir /path_to_data_dir/squad/v1.1
+--work_dir /path_to_output_folder
+--bert_checkpoint /path_to_bert_checkpoint
+--amp_opt_level "O1"
+--batch_size 24
+--num_epochs 2
+--lr_policy WarmupAnnealing
+--lr_warmup_proportion 0.0
+--optimizer adam_w
+--weight_decay 0.0
+--lr 3e-5
+--do_lower_case
+
+If --bert_checkpoint is not specified, training starts from
+Huggingface pretrained checkpoints.
+
+To finetune Squad v1.1 on pretrained BERT large uncased on 8 GPU:
+python -m torch.distributed.launch --nproc_per_node=8 squad.py
+--amp_opt_level "O1"
+--data_dir /path_to_data_dir/squad/v1.1
+--bert_checkpoint /path_to_bert_checkpoint
+--batch_size 3
+--num_gpus 8
+--num_epochs 2
+--lr_policy WarmupAnnealing
+--lr_warmup_proportion 0.0
+--optimizer adam_w
+--weight_decay 0.0
+--lr 3e-5
+--do_lower_case
+
+On Huggingface the final Exact Match (EM) and F1 scores are as follows:
+Model	                EM      F1
+BERT Based uncased      80.59    88.34
+BERT Large uncased      83.88    90.65
+"""
+
+import argparse
+import os
+import sys
+import json
+import nemo
```

Comment:
Can you group nemo and nemo_nlp imports?
Also a huge plus if your imports are alphabetical within a section. Eg all python-native libraries together in alphabetical order, then all pip packages in alphabetical order, then nemo in alphabetical order

Reply:
I changed the source branch accordingly